src/main/java/br/eti/kinoshita/testlinkjavaapi/model/TestPlan.java

```diff
@@ -24,6 +24,8 @@
 package br.eti.kinoshita.testlinkjavaapi.model;
 
 import java.io.Serializable;

```

Comment:
Excellent work! Will bump the serialVersionUID too as the TestPlan constructor and properties changed. Kudos!